src/utils/isPrevMonth.js

```diff
@@ -0,0 +1,6 @@
+import moment from 'moment';
+
+export default function isPrevMonth(a, b) {
+  if (!moment.isMoment(a) || !moment.isMoment(b)) return false;
+  return b.isSame(a.clone().subtract(1, 'month'), 'month');
```

Comment:
I've found `isSame` to be slow even when it's got a 'month' or 'day' or whatever argument. Maybe let's do:
```
return b.month() === a.clone().subtract(1, 'month').month();
```